test/mocha/new.js

```diff
@@ -2,7 +2,7 @@ var assert = require("assert");
 var uglify = require("../../");
 
 describe("New", function() {
-    it("Should attach callback parens after some tokens", function() {
+    it("Should attach callback parentheses in beautify mode", function() {
```

Comment:
Does the phrase "callback parentheses" occur in the ES5.1 spec or something? I am not familiar with it.